src/vendor/bcur/index.js

```diff
@@ -1,2 +1,2 @@
-export { encodeUR } from './encodeUR';
-export { smartDecodeUR, decodeUR, extractSingleWorkload } from './decodeUR';
+export { encodeUR } from "./encodeUR";
+export { smartDecodeUR, decodeUR, extractSingleWorkload } from "./decodeUR";
```

Comment:
Single quote and double quotes being used inconsistently. I'm a single-quoter for anything that isn't supposed to be readable text for the end user. Can we make it the prettier rule?

Reply:
It's only inconsistent because we didn't have prettier before. It's being added in files that I'm clicking "save" on for the first time (we should probably add a pre-commit hook tbh). As noted elsewhere, I'm preferring for consistency with other code bases we maintain. If we want to make a change to all our rules to be single quotes, I'm fine with that, but for now inconsistency across projects is a bigger friction point in my opinion.

Reply:
There is no consistency between code bases. Yellen is using single quotes.